README.md

```diff
@@ -4,10 +4,19 @@
   <a href="https://github.com/actions/setup-ruby"><img alt="GitHub Actions status" src="https://github.com/actions/setup-ruby/workflows/Main%20workflow/badge.svg"></a>
 </p>
 
-This action sets up a ruby environment for use in actions by:
+This action sets up Ruby for use in actions by adding to the `PATH` a version of Ruby already existing in the toolcache.
+The toolcache is the set of tools preinstalled in [virtual environments](https://github.com/actions/virtual-environments).
 
-- optionally installing a version of ruby and adding to PATH. Note that this action only uses versions of Ruby already installed in the cache. The action will fail if no matching versions are found.
-- registering problem matchers for error output
+This action currently only supports versions of Ruby in the toolcache.
+See [this list](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md) for the available Ruby versions.
```

Comment:
Instead of having it as a follow sentence below (a detail) the minor version point should be pushed up to be the main point.  e.g. the first sentence that points to the versions should say "bind to a minor version for versions install on the virtual machines" and then links etc.

Reply:
Done